.github/ci-scripts/bash/functions-for-build

```diff
@@ -233,6 +233,40 @@ frp_build_and_push() {
   set +e
 }
 
+get_base_branch_commit_from_pr_number() {
+  pr_number="$1"
+  base_branch_commit_of_root_pr=$(curl -L \
+  -H "Accept: application/vnd.github+json" \
+  -H "Authorization: Bearer ${GITHUB_TOKEN}" \
+  -H "X-GitHub-Api-Version: 2022-11-28" \
+  "${GH_API_URL}/repos/${REPOSITORY}/pulls/${root_pr_number}"| jq -r '.merge_commit_sha')
```

Comment:
$1 is `pr_number` but in API using `root_pr_number`

Reply:
the variable is getting declared in get_root_pr_tag, and looks it can be passed without providing it as argument.
removing pr_number and root_pr_number from function argument